Docker version should be 17.06 in the Powershell script

shijiong opened the pull request:
1. The output from iotedgectl.exe tool in green (not red) in the remote PowerShell window.
2. Docker version should be "17.06", not "17.09" in the Powershell script, which can be downloaded by "https://aka.ms/iotedgewin".

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

shijiong committed:
Docker version should be 17.06 in the Powershell script

1. The output from iotedgectl.exe tool in green (not red) in the remote PowerShell window.
2. Docker version should be "17.06", not "17.09" in the Powershell script, which can be downloaded by "https://aka.ms/iotedgewin".
